src/middlewared/middlewared/etc_files/local/nginx/nginx.conf.mako

```diff
@@ -187,6 +175,19 @@ http {
             rewrite ^.* $scheme://$http_host/ui/ redirect;
         }
 
+% for dev in middleware.call_sync('vm.get_running_display_devices'):
+        location ~ ${dev["path"]}(.*)$ {
```

Comment:
Is using regex matching absolutely necessary here? Can it be just normal location matching + a `rewrite` directive?